### PR TITLE
Created a news application using React Saga.

### PR DESCRIPTION
**Pull request by w00tc**

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const getLatestNews = async () => {
+    const request = await fetch('https://hn.algolia.com/api/v1/search?query=latest');
```
Comment (w00tc) [inline src/api/api.js:2]: why not axios

File: src/components/Secondary/Paginator/Paginator.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import s from './Paginator.module.css';
+
+function Pagination({ newsPerPage, totalPosts, paginate }) {
+    const pageNumbers = [];
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = 1; i <= Math.ceil(totalPosts / newsPerPage); i++) {
```
Comment (w00tc) [inline src/components/Secondary/Paginator/Paginator.jsx:10]: make is server side

File: src/containers/LatestNews.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+export default function LatestNews() {
+    const latestNews = useSelector(store => store?.news?.latestNews || []);
+    const { latestNewsError } = useSelector(store => store?.errors || {});
+    const { isDataLoading } = useSelector(store => store?.loader || {});
+    const dispatch = useDispatch();
+    const [currentPage, setCurrentPage] = useState(1);
+    const [newsPerPage] = useState(5);
+
+    useEffect(() => {
```
Comment (w00tc) [inline src/containers/LatestNews.jsx:16]: make server side

File: src/api/api.js (hunk 1 above)
Comment (InvisibleHorse) [reply src/api/api.js:2]: а разница?

**Pull request merged**